client/gutenberg/extensions/visited/components/edit.jsx

```diff
@@ -0,0 +1,80 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { memoize } from 'lodash';
+import { Notice, PanelBody, RadioControl, TextControl } from '@wordpress/components';
+import { Component, Fragment } from '@wordpress/element';
+import { InnerBlocks, InspectorControls } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import { sprintf } from '@wordpress/i18n';
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { CRITERIA_AFTER, CRITERIA_BEFORE } from '../constants';
+
+const getRadioOptions = memoize( threshold => [
+	{ label: sprintf( 'Show after %d views', threshold ), value: CRITERIA_AFTER },
+	{ label: sprintf( 'Show before %d views', threshold ), value: CRITERIA_BEFORE },
```

Comment:
`'Show after %d views'` and `'Show before %d views'` need to be wrapped in `__()` translation function.

Reply:
Gotcha, good to know!